management/client.go

```diff
@@ -240,6 +277,28 @@ func (m *ClientManager) Delete(id string, opts ...RequestOption) error {
 	return m.Request("DELETE", m.URI("clients", id), nil, opts...)
 }
 
+// CreateCredential creates an client application's client credential.
+func (m *ClientManager) CreateCredential(id string, credential *Credential, opts ...RequestOption) error {
```

Comment:
Shouldn't this return the created credential? See https://github.com/auth0/node-auth0/pull/734/files#diff-8f2f9669ed871610c0349f76ff7819e843d36da69a7818c90de89c3036ca3cddR65.

Reply:
Nevermind, it receives a pointer, so it should update in-place.